src/window.rs

```diff
@@ -156,6 +158,36 @@ impl Window {
             .or_else(|_| create_gl_window(window_builder, &event_loop, true))?;
         window.show();
 
+        #[cfg(not(any(windows, target_os = "macos")))]
+        {
+            if window_config.decorations() == Decorations::Transparent && event_loop.is_wayland() {
+                let bg = colors.primary.background;
+                let mut scale = 382.5 / (f32::from(bg.r) + f32::from(bg.g) + f32::from(bg.b));
+                scale /= 3.;
+                let scaled_bg = [0xFF, (f32::from(bg.r) * scale) as u8, (f32::from(bg.g) * scale) as u8, (f32::from(bg.b) * scale) as u8];
+                scale *= 1.5;
+                let vscaled_bg = [0xFF, (f32::from(bg.r) * scale) as u8, (f32::from(bg.g) * scale) as u8, (f32::from(bg.b) * scale) as u8];
```

Comment:
Lines should be shorter than 100 characters. Generally following rustfmt is a good idea, even though it's not possible to run it on the whole codebase yet.